app/pb_kits/playbook/pb_gauge/docs/_gauge_min_max.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Gauge } from '../../'
+
+const data = [{
+  name: 'Rating', value: 4.5,
+}]
+
+const GaugeMinMax = () => (
+  <div>
+    <Gauge
+        chartData={data}
+        id="gauge-min-max"
+        // max defaults to 100
+        max={5}
+        // min defaults to 0
```

Comment:
Could we maybe add this info to the doc example text itself so that it is present and viewable in the browser?